FAQ.md

```diff
@@ -1,6 +1,7 @@
-# Frequently Asked Questions (FAQ)
+**Frequently Asked Questions (FAQ)**
```

Comment:
This change makes the text look bold, instead of a separate section (which it was before). Could you please revert this change?

Reply:
This will make that line look too big and bold too.

Reply:
As before, it's a separate section now.